config/config.template.toml

```diff
@@ -14,31 +14,35 @@ log_file = "logs/unit_tests.log"
 
 # GELF logging output
 [logging.gelf]
-enabled = false
+enabled = true
```

Comment:
This should be default False, since unless you actually spin up a graylog cluster and configure it, this will produce needless network traffic and overhead.